editor/plugins/org.fusesource.ide.projecttemplates/META-INF/MANIFEST.MF

```diff
@@ -46,14 +46,17 @@ Require-Bundle: org.eclipse.ui;bundle-version="3.106.0",
  org.fusesource.ide.camel.validation;bundle-version="9.1.0",
  org.eclipse.m2e.archetype.common;bundle-version="1.6.3",
  org.eclipse.jdt.ui;bundle-version="3.11.2",
- org.eclipse.m2e.wtp;bundle-version="1.2.2"
+ org.eclipse.m2e.wtp;bundle-version="1.2.2",
+ org.fusesource.ide.preferences;bundle-version="9.1.0"
 Bundle-ActivationPolicy: lazy
 Bundle-Activator: org.fusesource.ide.projecttemplates.internal.ProjectTemplatesActivator
 Export-Package: org.fusesource.ide.projecttemplates.adopters,
  org.fusesource.ide.projecttemplates.adopters.util,
  org.fusesource.ide.projecttemplates.impl.medium,
  org.fusesource.ide.projecttemplates.impl.simple,
+ org.fusesource.ide.projecttemplates.internal,
```

Comment:
exporting the internal package? if required by some test plugin, it is better to use x-friends to reduce visibility (even if we are not currently very strict everywhere...)